umbraco/Umbraco-CMS
# 15209 media picker shows incorrect date

### Prerequisites


If there's an existing issue for this PR then this fixes [15209](https://github.com/umbraco/Umbraco-CMS/issues/15209)

### Description


This pull request contains the following changes.

- In mediapicker.controller.js, remove the 'Z' character representing UTC+0 timezone to ensure that "Last Updated" date for media picker shows time in local user's time zone.

Steps to test (taken from main issue body): 

1. Ensure SQL Server is Database (not SQL Lite)
2. Upload a file to "Media"
3. Create a document type, with "Allow as Root" enabled and "Media Picker" property.
4. In "Content", create item and click the "Add" button.
5. In the dialog choose list view
6. Notice that date is as expected as per your local time zone.
7. (Optional) Repeat 1-6 for other database types to ensure that date is still correct for timezone.

**A note about this change:** From debugging, it looks like the last update date time is correct, however the 'Z' is appended because I think that the user timezone is based in UTC+0 as there are many calls to DateTime.UtcNow in UserRepository.cs and other database code. The fix above will mask this issue but really, any media uploaded will still technically have a last updated string that is in UTC+0.
